code/datums/extensions/encumbrance.dm

```diff
@@ -4,7 +4,7 @@
 
 /datum/extension/updating/encumbrance
 	flags = EXTENSION_FLAG_IMMEDIATE
-	expected_type = /mob/living
```

Comment:
why are you messing with my new extension? Encumbrance is only for mob equipment, its not to be repurposed for anything else